src/contacts.jl

```diff
@@ -29,15 +29,18 @@ struct ContactPoint{N}
     ρ::SVector{N, Variable} # basis vector multipliers
     force_local::FreeVector3D{SVector{3, Variable}} # contact force expressed in contact point's normal-aligned frame
     wrench_world::Wrench{Variable}
-    position::Any # The position, normal, and μ fields are not concretely typed,
-    normal::Any   # but these fields are not used in any of the contact handling algorithms.
-    μ::Any        # Instead, they are just stored for debugging purposes.
+    position::Any
+    normal::Any
+    μ::Any
+    toroot::Any
```

Comment:
Undo removal of comments or just remove fields?

Reply:
It actually turned out to be very useful to store these fields in the contact so that I can reference them in the LCP setup. The fact that they're untyped doesn't actually matter, which is really nice. I'll add a more helpful comment.